### PR TITLE
Bound live transcript memory growth

### PR DESCRIPTION
**Pull request by ComputelessComputer**
Render active captures directly from live segments, remove per-segment transcript subscriptions, and isolate volatile render caches. Adds full-descendant regression coverage for 500 segments and active-to-settled rendering.

---

> [!NOTE]
> **Medium Risk**
> Touches core live recording, transcript integrity, and caching during capture; regressions could drop or duplicate words or mis-label speakers, though coverage is extensive.
> 
> **Overview**
> Introduces **per-session capture generations** in the listener store so each recording gets a stable identity from start through finalizing, with retries and attach/detach edge cases covered by new tests.
> 
> **Active transcript rendering** now uses a frozen native-render **baseline** keyed by capture generation (`volatile` vs `settled` React Query caches), so live word and speaker-assignment writes no longer trigger a full re-segment on every SQLite update. The UI threads `captureGeneration` into the viewer; only the **last** transcript row stays active with live segments while earlier rows render settled.
> 
> **Live/persisted merge** gains reconciliation helpers: preserve prefixes when the live tail overlaps persisted segments, drop stale frozen words when IDs rotate, and apply current assignments to visible live segments in JS. Speaker labels are computed once per transcript list instead of per segment header.
> 
> Docs/changelog note long-session responsiveness; QA skill adds Keychain **Computer Use** steps. CloudSync integration test pool-drain timeouts increase from 250ms to 1500ms.
> 
> <sup>Reviewed by [Cursor Bugbot](https://cursor.com/bugbot) for commit 5885ee2defa78e1866086f3c8cdaccc7e8f71f80. Bugbot is set up for automated code reviews on this repo. Configure [here](https://www.cursor.com/dashboard/bugbot).</sup>

### CODE REVIEW

File: apps/desktop/src/session/components/note-input/transcript/renderer/transcript.tsx [hunk 1]
```diff
@@ -39,20 +43,111 @@ export function RenderTranscript({
...
+        segments={liveSegments}
+        scrollElement={scrollElement}
+        transcriptId={transcriptId}
+        shouldScrollToEnd={isLastTranscript && shouldScrollToEnd}
+        currentMs={currentMs}
+        seek={seek}
+        startPlayback={startPlayback}
+        audioExists={audioExists}
+      />
+    );
```
[the remark this answers is not in the document]
Comment (ComputelessComputer) [reply apps/desktop/src/session/components/note-input/transcript/renderer/transcript.tsx]: Fixed in 2de3174f18. Active rendering now reconciles the persisted SQLite baseline with live segments at word granularity, preserves recovered prefixes, handles replacement IDs, and keeps one bounded native-render cache key. Recovery, overlap, remount, and large-input regressions pass, along with the full 2,251-test desktop suite.

File: apps/desktop/src/session/components/note-input/transcript/renderer/data-hooks.ts [hunk 2]
```diff
@@ -18,27 +20,53 @@ export function useRenderedTranscriptSegments(transcriptId: string): Segment[] {
...
-      if (!request) {
+      if (!activeBaselineRequest) {
         return [];
       }
 
-      return renderTranscriptSegments(request);
+      return renderTranscriptSegments(activeBaselineRequest);
     },
-    enabled: !!request,
+    enabled: !!activeBaselineRequest,
```
[the remark this answers is not in the document]
Comment (ComputelessComputer) [reply apps/desktop/src/session/components/note-input/transcript/renderer/data-hooks.ts:79]: Fixed in 9956a2b14d. Each WebView now assigns a monotonic per-session capture generation across start requests, native events, and snapshots, then releases only that session on stop or failed start. The volatile query key rotates between captures while remaining stable across same-capture tab remounts. Regressions cover forwarded standalone starts, concurrent finalization, event ordering, and a real QueryClient unmount/remount; the full desktop suite passes 2,259 tests.
